oioioi/programs/templates/programs/report-signal-hint.html

```diff
@@ -0,0 +1,78 @@
+{% load i18n %}
+
+<div class="alert alert-info">
+    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-info-circle-fill" viewBox="0 0 16 16"><path d="M8 16A8 8 0 1 0 8 0a8 8 0 0 0 0 16m.93-9.412-1 4.705c-.07.34.029.533.304.533.194 0 .487-.07.686-.246l-.088.416c-.287.346-.92.598-1.465.598-.703 0-1.002-.422-.808-1.319l.738-3.468c.064-.293.006-.399-.287-.47l-.451-.081.082-.381 2.29-.287zM8 5.5a1 1 0 1 1 0-2 1 1 0 0 1 0 2"></path></svg>
+    <br>
+    {% for signal in signals_to_explain %}
+        {% if signal == 6 %}
+            {% blocktrans %}
+                <b>Most common causes of the SIGABRT signal (signal 6):</b>
+                <br>
```

Comment:
is there any reason not use <UL>?

Reply:
You're right. It looks better, too. I fixed this in the newest commit.